src/mgmt/pool.pb-c.c

```diff
@@ -1317,7 +1317,7 @@ static const ProtobufCFieldDescriptor mgmt__pool_create_req__field_descriptors[1
     offsetof(Mgmt__PoolCreateReq, faultdomains),
     NULL,
     NULL,
-    0,             /* flags */
+    0 | PROTOBUF_C_FIELD_FLAG_PACKED,             /* flags */
```

Comment:
these changes should not be submitted as they are just due to pb tool version differences not due to intended proto changes